doc/sphinxext/gallery_generator.py

```diff
@@ -39,8 +39,8 @@ def execfile(filename, globals=None, locals=None):
 
 .. image:: {img_file}
 
-**Python source code:** :download:`[download source: {fname}]<{fname}>`
-**API documentation:** {api_name} 
+|   **Python source code:** :download:`[download source: {fname}]<{fname}>`
+|   **API documentation:** {api_name} 
```

Comment:
I would use `\n` before `**API documentation**...`

Reply:
I tried to use this, but it doesn't seem to be supported in `rsT`.

Reply:
Oh, I would have thought that given that the file is written by the python script the newline would be directly present in the rst file, good to know. Then leaving a blank line between two links should do the trick

Reply:
Yes, that does the trick. :)